common/src/path-utils.test.ts

```diff
@@ -0,0 +1,80 @@
+import { filterPaths } from "./path-utils";
+
+describe('path utils', () => {
+
+    it.each([
+        // #region cases without negation
```

Comment:
minor: i think it's better to split them with `describe` instead of `// #region`

Reply:
Done, thanks